### DISCUSSION TITLE
Is the "Screen Edges" feature not supposed to honor the chosen layouts?

### DISCUSSION THREAD

**Discussion by tspaltenberger** [domferr/tilingshell#192] [Q&A] [answered]
I'm on Gnome 47.1/Tiling Shell 15.0. 

I'm not 100% sure if this was different before (therefore I did not raise an issue straightaway), but I just realized that in my case, the "Screen Edges" functionality is (no longer?) honoring the defined layouts. Instead, it is always splitting the screen in half (or in quarters, if the window is dragged in the "quarter tiling activation area"). 

Am I missing something, am I doing something wrong or is this a bug?

![grafik](https://github.com/user-attachments/assets/5d66d1f1-8e5a-455a-a511-a81f5d2cba71)
![grafik](https://github.com/user-attachments/assets/fb63d4f7-135a-4e39-8a35-5c52ec9f8f4f)
[Screen Edge Tiling recording](https://github.com/user-attachments/assets/b12bda32-a6ac-4b8d-814d-9137c227fb23)
[tilingshell-settings.txt](https://github.com/user-attachments/files/17789767/tilingshell-settings.txt)

**Comment by domferr** [comment, marked as answer]
Hey thank you for opening this! Yeah, screen edges were and are not following the currently selected layout. As you noticed, they always follow a "quarter tiling" layout.
